extra/alerts/alert_snmp.sh.sample

```diff
@@ -116,6 +93,24 @@ is_in_list() {
     return 1
 }
 
+# Usage: send_pacemaker_trap <node> <rsc> <op> <desc> <status> <rc> <target-rc> <attr-name> <attr-value>
+# Callers should specify all values, even if not needed for a particular alert type.
+send_pacemaker_trap() {
+    "${trap_binary}" -v "${trap_version}" ${trap_options} \
+        -c "${trap_community}" "${CRM_alert_recipient}" "" \
+        "PACEMAKER-MIB::pacemakerNotificationTrap" \
+        "PACEMAKER-MIB::pacemakerNotificationNode" s "$1" \
+        "PACEMAKER-MIB::pacemakerNotificationResource" s "$2" \
+        "PACEMAKER-MIB::pacemakerNotificationOperation" s "$3" \
+        "PACEMAKER-MIB::pacemakerNotificationDescription" s "$4" \
+        "PACEMAKER-MIB::pacemakerNotificationStatus" i "$5" \
+        "PACEMAKER-MIB::pacemakerNotificationReturnCode" i "$6" \
+        "PACEMAKER-MIB::pacemakerNotificationTargetReturnCode" i "$7" \
+        "PACEMAKER-MIB::pacemakerNotificationAttributeName" s "$8" \
+        "PACEMAKER-MIB::pacemakerNotificationAttributeValue" s "$9" \
+        "${hires_timestamp}" 2>&1
```

Comment:
This double quotation seems unnecessary.
If it exists, traps can not be sent.

```
${hires_timestamp} 2>&1
```

Reply:
Thank you! I should have caught that. :-/ I've updated the request with this fix.